### ISSUE TITLE
Error: E216: No such group or event: VimSuspend,VimResume * call s:Heartbeat()

### ISSUE THREAD

**Issue by maksze** [ActivityWatch/aw-watcher-vim#16]
When i start VIM 8.2.148 i get error
```
Error detected while processing /home/maksze/.vim/bundle/aw-watcher-vim/plugin/activitywatch.vim:
line  150:
E216: No such group or event: VimSuspend,VimResume * call s:Heartbeat()
```

**Comment by johan-bjareholt** [comment]
Oh, that was my mistake. https://github.com/ActivityWatch/aw-watcher-vim/commit/75d744ec4487c8956fea544f17b9af1ad808c0f3

I added them assuming that they existed in both neovim as well as vim but apparently not. Removed them again, verified that it works with normal vim.

Fixed here https://github.com/ActivityWatch/aw-watcher-vim/commit/326decbf61271c0716be743b9fdf230979f9d4b1

Thanks for reporting!
